using-webauthn-mfa-in-command-line.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Using WebAuthn multifactor authentication in command line
+url: /using-webauthn-mfa-in-command-line
+previous: /using-mfa-in-command-line
+next: /using-otp-mfa-in-command-line
+---
+<em class="t-gray">How to use WebAuthn MFA with gem CLI.</em>
+
+Multifactor authentication (MFA) using WebAuthn works by using a
+removable hardware token or touch biometric / facial biometric capabilities built
+into your phone or computer. This is distinct from MFA based on typing or copying
+a code generated by an authentication app or password manager, called OTP. For
+OTP MFA see "[Using OTP multifactor authentication in command line](/ using-otp-mfa-in-command-line)".
+
+When you have enabled WebAuthn MFA we will ask you to perform authentication for
+`gem signin`. Check "[Setting up WebAuthn multifactor authentication](/setting-up-webauthn-multifactor-authentication)"
+for guidance on setting up WebAuthn MFA.
+
+    Enter your RubyGems.org credentials.
+    Don't have an account yet? Create one at https://rubygems.org/sign_up
+    Email:   gem_author@example
+    Password:
+
+    [snip of API key setup]
+
+    You have enabled multi-factor authentication.
+    Please visit http://localhost:3000/webauthn_verification/<random>?port=49385
+    to authenticate via security device.
```

Comment:
Should this be updated with the newest message from https://github.com/rubygems/rubygems/pull/6523?
Perhaps we should add a blurb that we default to webauthn instead of otp.

Reply:
I'll update it.